src/SWP/Bundle/RuleBundle/spec/Form/Type/RuleTypeSpec.php

```diff
@@ -28,17 +29,17 @@
  */
 final class RuleTypeSpec extends ObjectBehavior
 {
-    function it_is_initializable()
+    public function it_is_initializable()
```

Comment:
public is by default and it's good practice to define it even if it's default type. And it was fixed automaticaly by my phpcs fixer.